pkg/autodiscovery/autoconfig.go

```diff
@@ -332,6 +332,9 @@ func (ac *AutoConfig) pollConfigs() {
 			case <-ac.health.C:
 			case <-ac.configsPollTicker.C:
 				// check if services tags are up to date
+				servicesToRefresh := []listeners.Service{}
```

Comment:
`nit`: slices can be initialised to `nil` with no allocation by using the following syntax:

`var servicesToRefresh []listeners.Service`

appending/ranging on a `nil` slice is supported